R/remove.missing.levels.lm.R

```diff
@@ -0,0 +1,77 @@
+#' @title remove.missing.levels.lm
+#' @description Accounts for missing factor levels only present in test data
+#' but not in train data by setting levels to NA
+#'
+#' @importFrom stringi stri_split stri_replace_all
+#' @importFrom BBmisc warningf
+#'
+#' @param fit fitted model on training data
+#'
+#' @param test.data data to make predictions for
+#'
+#' @return data.frame with matching factor levels to fitted model
+#'
+#' @keywords internal
+#'
```

Comment:
Please have a look at the other Roxygen comments -- types are missing, formatting.